python/src/aac/plugins/gen_protobuf/gen_protobuf_impl.py

```diff
@@ -199,6 +198,19 @@ def _to_template_properties_dict(
         elif type(option_value) is bool:
             option_entry["value"] = str(option_value).lower()
 
+    # Format the names
+    name = _sanitize_string_to_pascal_case(name)
+
+    # Format the enum values
+    enums = [_sanitize_string_to_snake_case(enum) for enum in enums]
+    enums = [enum.upper() for enum in enums]
```

Comment:
You might be able to condense this with
```suggestion
    enums = [_sanitize_string_to_snake_case(enum).upper() for enum in enums]
```